cmd/integration/commands/state_domains.go

```diff
@@ -167,7 +167,7 @@ var purifyDomains = &cobra.Command{
 		if purifyOnlyCommitment {
 			purificationDomains = []kv.Domain{kv.CommitmentDomain}
 		} else {
-			purificationDomains = []kv.Domain{kv.AccountsDomain, kv.StorageDomain /*"code",*/, kv.CommitmentDomain, kv.ReceiptDomain, kv.RCacheDomain}
+			purificationDomains = []kv.Domain{kv.AccountsDomain, kv.StorageDomain /*"code",*/, kv.CommitmentDomain}
```

Comment:
Why this change?

Reply:
https://github.com/erigontech/erigon/pull/15049